orchestrator/careplancontributor/service.go

```diff
@@ -317,7 +457,7 @@ func (s Service) withSessionOrBearerToken(next func(response http.ResponseWriter
 			next(response, request)
 			return
 		}
-		http.Error(response, "no session found", http.StatusUnauthorized)
+		http.Error(response, "no session or bearer token found", http.StatusUnauthorized)
```

Comment:
static bearer token is probably not something we want to keep, so I'd like to keep the original error message.